.github/workflows/test.yml

```diff
@@ -120,17 +120,29 @@ jobs:
       - name: Build
         run: |
           foreach($target in ($env:BUILD_TARGETS).Split(',')) {
-            echo $target
+            echo "::group::Build for $target target"
             cargo build $(if($target -ne 'default') {"--target=$target"} ) 
+            echo "::endgroup::"
```

Comment:
Should this line be after the next `if` branch? `echo` always succeeds, which probably means we cannot detect build/test errors.

Reply:
I think you're right. Thanks for catching this!

Reply:
I just found this while running the tests that should fail :)

https://github.com/extendr/extendr/issues/129#issuecomment-761350724

Sure, I'll create a pull request for this.